### ISSUE TITLE
[ FR ] Limit hotkeys to vim normal mode

### ISSUE THREAD

**Issue by Gedeon23** [moolmanruan/obsidian-sequence-hotkeys#5]
As the title says it would be nice to limit hotkeys to vim normal mode so they don't interfere with typing

**Comment by moolmanruan** [comment]
Hi @Gedeon23, TIL that Obsidian supports a built-in Vim mode 😄 I'll have a look and see what I can do.

**Comment by moolmanruan** [comment]
I've asked for some help on Discord (https://discord.com/channels/686053708261228577/840286264964022302/982661534654292038) and it looks like one can determine if Vim mode is active, but it's not clear whether the API exposes a way to get the mode Vim is currently in.

**Comment by moolmanruan** [comment]
Hi @Gedeon23, I haven't found any other info about the API to help out with this issue yet. As a workaround you could probably use chords that include modifier keys. I assume that you are trying to use normal keys for shortcuts from the request.

**Comment by Gedeon23** [comment]
Yes I am. I'll probably look into the CodeMirror 6 api one of these days to see if it exposes the vim mode somehow. Someone on discord told me the entire CodeMirror api is available.

**Comment by huyz** [comment]
I have a related problem: it seems that my hotkeys only work in insert mode, and not normal mode.
I tried `⌥T` and `⌃S` as chord prefixes. What are you all using?

**Comment by moolmanruan** [comment]
Hey @huyz, I haven't looked into getting the plugin working with vim mode at the moment.

**Comment by huyz** [comment]
@Gedeon23 After trying out different chord key plugins, I realized that the only one that works in vim normal mode is https://github.com/esm7/obsidian-vimrc-support/

**Comment by moolmanruan** [comment]
Thanks @huyz, will have a look at that plugin a bit and see if I can learn something from it.
